### ISSUE TITLE
invalid FO output caused by handling of <byline>?

### ISSUE THREAD

**Issue by dasch124** [TEIC/Stylesheets#252]
Hi,
transforming this TEI document

~~~~
<TEI xmlns="http://www.tei-c.org/ns/1.0">
   <teiHeader>
      <fileDesc>
         <titleStmt>
            <title>Title</title>
         </titleStmt>
         <publicationStmt>
            <p>Publication Information</p>
         </publicationStmt>
         <sourceDesc>
            <p>Information about the source</p>
         </sourceDesc>
      </fileDesc>
   </teiHeader>
   <text>
      <body>
         <p>Some text here.</p>
         <byline><persName>Me</persName></byline>
      </body>
   </text>
</TEI>
~~~~

 with the current Stylesheets (using Oxygen Add-On version 6.1.1.) to PDF results in an error: 

> The process 'Apache FOP' ended with code: 1.
The error was: 
ERROR - net.sf.saxon.trans.XPathException: org.apache.fop.fo.ValidationException: "fo:inline" is not a valid child of "fo:flow"! (Siehe Position 1:-1)

Evidently this is caused by `<persName>` being a child of `<byline>`, which results in the following (invalid) FO markup:

~~~~
<flow flow-name="xsl-region-body" font-family="Times" font-size="10pt">
     <block line-height="1">Some text here.</block>
     <inline font-style="normal" font-weight="normal">Me</inline>
</flow>
~~~~~

I (wildly) guess this is caused by the template that matches tei:byline in common_core.xsl (line 725) calling  the `makeSpan` template which is implemented as `<xsl:apply-templates/>` in `fo_core.xsl`, line 952. 

I suppose it would be fair that `<byline>` created a `<block>` since its model is `model.pLike.front`. But my knowledge of the Stylesheets is very limited, so I didn't dare to submit a pull request. (I can, of course, offer to do so, if that solution seems sensible.)

Thank you for looking into this!

**Comment by martindholmes** [comment]
I've added a fix in commit 8b832d9 which handles your test case but doesn't change results for the instances of byline in our test suite. Are you able to test the development branch of the Stylesheets with your original source document? If so, let me know whether this solves the problem for you.

Many thanks for a clear bug report with a nice simple test case.

**Comment by martindholmes** [comment]
No response from the user, but I think this is fixed so I'm closing it.
